compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4900,15 +4903,15 @@ template <uint32_t numberOfBits>
 TR::Register *
 genericLoad(TR::Node * node, TR::CodeGenerator * cg, TR::MemoryReference * tempMR, TR::Register * srcRegister)
    {
-   bool nodeSigned = node->getType().isInt64() || !node->isUnsignedLoad();
+   bool nodeSigned = node->getType().isInt64() || !node->isZeroExtendedAtSource();
 
    if (node->getType().isAddress() || node->getType().isAggregate()) // o- and a-type and PLX-Fixed24/Fixed8 always unsigned
       nodeSigned = false;
 
-   if (numberOfBits>32 || numberOfBits==31 || node->force64BitLoad())
-      return genericLoadHelper<numberOfBits, 64, MemReg>(node, cg, tempMR, srcRegister, nodeSigned, node->couldIgnoreExtend());
+   if (numberOfBits > 32 || numberOfBits == 31 || node->isExtendedTo64BitAtSource())
+      return genericLoadHelper<numberOfBits, 64, MemReg>(node, cg, tempMR, srcRegister, nodeSigned, false);
    else
-      return genericLoadHelper<numberOfBits, 32, MemReg>(node, cg, tempMR, srcRegister, nodeSigned, node->couldIgnoreExtend());
+      return genericLoadHelper<numberOfBits, 32, MemReg>(node, cg, tempMR, srcRegister, nodeSigned, false);
```

Comment:
Why is the last parameter `false` in both cases?

Reply:
`couldIgnoreExtend()` has been deprecated. There were no references to `setCouldIgnoreExtend` in the entire codebase as well as any downstream projects. It mappend to `false` in all cases.